db/schema.rb

```diff
@@ -37,10 +37,8 @@
 
   create_table "notifications", force: :cascade do |t|
     t.integer "user_id"
-    t.string  "kind"
```

Comment:
This changes are right, the last migration removes this fields, but the schema was not updated.